lib/moon/components/menu_item/checkbox.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Moon.Components.MenuItem.Checkbox do
+  @moduledoc false
+
+  use Moon.StatelessComponent
+
+  import Moon.Components.Checkbox, only: [fake_checkbox: 1]
+
+  prop(is_selected, :boolean, from_context: {Moon.Components.MenuItem, :is_selected})
+
+  def render(assigns) do
+    ~F"""
+    <span class="flex w-6 h-6 justify-center items-center">
+      <.fake_checkbox selected={@is_selected} />
```

Comment:
Why you just did not use Checkbox component with value=true?

Reply:
i've used just the look and have skipped all the form logic

Reply:
also because it was already separated inside the module